doc/oidc/auth0.md

```diff
@@ -0,0 +1,211 @@
+[<- Back to Index](../index.md)
+
+# Auth0 OIDC Setup Guide
+
+This guide provides basic steps to authenticate and authorize SQL LRS (running locally on your machine) against the [Auth0 Cloud Identity Provider](https://auth0.com/). This guide is provided for example purposes only and may not produce an environment suitable for production usage.
+
+### Getting Started
+
+Make an account on [Auth0](https://auth0.com) and log in to the administrative console. Make sure you have the [latest release of SQL LRS](https://github.com/yetanalytics/lrsql/releases) downloaded to your machine and unzipped. For this guide you'll need access to a command line and an application for editing JSON files.
+
+### Resource Server (API) Setup
+
+#### Create an API
+
+To enable token authentication to SQL LRS you'll need to configure Auth0 to use it as an OAuth 2.0 Resource Server. To do this, log in to the [Auth0 Dashboard](https://manage.auth0.com/) and navigate to Applications > APIs. Click "+ Create API":
```

Comment:
Ok kind of confused by this wording since the main OIDC page says the SQL LRS server _itself_ is the OAuth 2.0 resource server. Is this particular scenario offloading that particular task away from SQL LRS onto Auth0? It could be clearer if you mentioned this sort of offloading/distinction.

Reply:
what if I just replace "it" with "the LRS"

Reply:
Ah yes that would be much clearer. Right now I'm interpreting "it" as "the Auth0 server."

Reply:
https://github.com/yetanalytics/lrsql/pull/271/commits/52d624dc2e12a265d854d070c10d6c81ce43f208